sympy/physics/vector/frame.py

```diff
@@ -35,11 +35,19 @@ class CoordinateSym(Symbol):
     ========
 
     >>> from sympy.physics.vector import ReferenceFrame, CoordinateSym
+    >>> from sympy import symbols
     >>> A = ReferenceFrame('A')
+    >>> B, C = symbols('B C', cls=ReferenceFrame, indices=('1', '2', '3'))
     >>> A[1]
     A_y
+    >>> B[0]
+    B_x
     >>> type(A[0])
     <class 'sympy.physics.vector.frame.CoordinateSym'>
+    >>> type(A[0]) == type(B[0])
+    True
+    >>> B.indices
+    ('1', '2', '3')
```

Comment:
I think this would be better as a separate example under ReferenceFrame or in the files under `docs/` that show advanced features.

Reply:
Thanks for the review @moorepants! I will close this PR, and open a new PR with an update to just the ReferenceFrame docstring, and the `docs/`.
Just a question, how should I separate examples? Could I just have an empty line between the previous example and mine, with a single line to introduce the example? Something like this:

```
>>> from sympy.physics.vector import ReferenceFrame, CoordinateSym
>>> A = ReferenceFrame('A')
>>> A[1]
A_y
>>> type(A[0])
<class 'sympy.physics.vector.frame.CoordinateSym'>
>>> a_y = CoordinateSym('a_y', A, 1)
>>> a_y == A[1]
True

# Initialising Multiple Reference Frames:
>>> from sympy.physics.vector import ReferenceFrame, CoordinateSym
>>> from sympy import symbols
>>> A = ReferenceFrame('A')
>>> B, C = symbols('B C', cls=ReferenceFrame, indices=('1', '2', '3'))
>>> B[0]
B_x
>>> type(A[0]) == type(B[0])
True
>>> B.indices
('1', '2', '3')
>>> c_y = CoordinateSym('c_y', C, 1)
>>> c_y == C[1]
True
```

Reply:
Yes, that can work. Make sure to use a sentence or two to explain the example you are introducing. And don't put it in the CoordinateSystem class docstring. Put it in the ReferenceFrame docstring.

Reply:
Thanks! I'll have it sorted out.